### PR TITLE
this is the big import of fluent-ui-fabric repo

### PR DESCRIPTION
**Pull request by kenotron**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This is our initial run of putting fluent-ui-react code inside the fabric repo. We'll use this PR to test out CI and practice publishing, etc. We'll continue to refine this PR until it is ready.

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11897)

### CODE REVIEW

**Comment by khmakoto** [comment]
Why are all size auditor and perf test results changing?

**Comment by JasonGore** [comment]
> Why are all size auditor and perf test results changing?

I suspect it's because React is bombing on hooks due to there being multiple instances of it. We have a LOT of work to do on this PR. It really should have been made in draft state to communicate that more obviously.

File: packages/utilities/src/asAsync.test.tsx [hunk 1]
```diff
@@ -20,12 +20,14 @@ describe('asAsync', () => {
     const wrapper = mount(<AsyncThing />);
 
     expect(_loadCalled).toBe(true);
-    expect(wrapper.text()).toBeNull();
+    expect(wrapper.text()).toBeFalsy();
     expect(_resolve).toBeTruthy();
 
     _resolve(() => <div>thing</div>);
+    // console.log('text: ', wrapper.find('#root').text());
```
Comment (khmakoto) [inline packages/utilities/src/asAsync.test.tsx]: Should we remove this?

Comment (kenotron) [reply packages/utilities/src/asAsync.test.tsx]: yes!

**Pull request merged**